src/runtime/pkg/govmm/qemu/qemu.go

```diff
@@ -1503,6 +1504,11 @@ func (vhostuserDev VhostUserDevice) QemuFSParams(config *Config) []string {
 	deviceParams = append(deviceParams, driver)
 	deviceParams = append(deviceParams, fmt.Sprintf("chardev=%s", vhostuserDev.CharDevID))
 	deviceParams = append(deviceParams, fmt.Sprintf("tag=%s", vhostuserDev.Tag))
+	queueSize := uint32(1024)
+	if vhostuserDev.QueueSize != 0 {
```

Comment:
Does this keep the minimum change(Not sure setting a default 1024 does matter )?

```golang
if vhostuserDev.QueueSize != 0 {
	deviceParams = append(deviceParams, fmt.Sprintf("queue-size=%d", vhostuserDev.QueueSize))
}
```

Reply:
As comment above, here uses 1024 to keep same with CLH. Qemu's default 128 is a bit small as FUSE uses a 256 pages limit:
https://github.com/torvalds/linux/blob/v6.0/fs/fuse/fuse_i.h#L39